app/controllers/api/invitations_controller.rb

```diff
@@ -5,7 +5,11 @@ def create
                                                      group: @group,
                                                      inviter: current_user,
                                                      message: invitation_form_params[:message])
-    respond_with_collection
+    if @invitations.empty? and email_addresses.any?
```

Comment:
Can I suggest the text:

'We weren't able to send invitations to those email addresses because those
people are in the group already.'

On Thu, 25 Feb 2016 at 14:52 James Kiesel <notifications@github.com> wrote:

> In app/controllers/api/invitations_controller.rb
> <https://github.com/loomio/loomio/pull/3014#discussion_r54038717>:
>
> > @@ -5,7 +5,11 @@ def create
> >                                                       group: @group,
> >                                                       inviter: current_user,
> >                                                       message: invitation_form_params[:message])
> > -    respond_with_collection
> > +    if @invitations.empty? and email_addresses.any?
>
> As it stands it shouldn't get to the server unless there are email
> addresses, since we disable the form. I'd be happy with a generic error
> message like
>
> 'We weren't able to send any invitations to the emails provided. They have
> been added to the group already.'
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/3014/files#r54038717>.
>